hack/jenkins/common.sh

```diff
@@ -283,9 +283,12 @@ set -x
 if test -f "${TEST_OUT}"; then
   rm "${TEST_OUT}" || true # clean up previous runs of same build
 fi
+
+make kic-base-image 
+
 touch "${TEST_OUT}"
 ${SUDO_PREFIX}${E2E_BIN} \
-  -minikube-start-args="--driver=${VM_DRIVER} ${EXTRA_START_ARGS}" \
+  -minikube-start-args="--driver=${VM_DRIVER} --docker-base-image=kic ${EXTRA_START_ARGS}" \
```

Comment:
should be added to our github actions too
I recommend adding a new job in the 3 jobs groups that are running before integraiton tests
currently they are build_minikube, unit_test, lint
we should add another job called -build build-base image-
and make the follwing integraiton jobs adds
needs: [build-minikube , build-base-image-